ios/RNSScreen.h

```diff
@@ -94,6 +94,7 @@ namespace react = facebook::react;
 @property (nonatomic) RNSScrollEdgeEffect leftScrollEdgeEffect;
 @property (nonatomic) RNSScrollEdgeEffect rightScrollEdgeEffect;
 @property (nonatomic) RNSScrollEdgeEffect topScrollEdgeEffect;
+@property (nonatomic) BOOL unstable_synchronousUpdatesEnabled;
```

Comment:
```suggestion
@property (nonatomic, readwrite) BOOL synchronousShadowStateUpdatesEnabled
```

we have many different "updates", I'd like to be more precise here, so that it is clear what this field does. 

Also, our native API is not stable and not really part of our library API, which is in JS. Therefore when doing similar things I omit the `ustable` / `experimental` prefix.

Reply:
https://github.com/software-mansion/react-native-screens/pull/3282/commits/0ffe03708d0e1c942d62ce58f6128e6e61730cd9